src/main/java/uk/gov/hmcts/ccd/ApplicationParams.java

```diff
@@ -86,15 +86,21 @@
     @Value("${definition.cache.max-idle.secs}")
     private Integer definitionCacheMaxIdleSecs;
 
-    @Value("${definition.cache.latest-version-ttl}")
+    @Value("${definition.cache.latest-version-ttl.secs}")
     private Integer latestVersionTTLSecs;
 
-    @Value("${definition.cache.jurisdiction-ttl}")
-    private Integer jurisdictionTTL;
+    @Value("${definition.cache.jurisdiction-ttl.secs}")
+    private Integer jurisdictionTTLSecs;
 
-    @Value("${user.cache.ttl.secs}")
+    @Value("${definition.cache.user-ttl.secs}")
```

Comment:
any reason for definition prefix for these?

Reply:
Reverted back the changes.